README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+#### (a)
+The algorithm performs a search of the list to find if there are any duplicate items
+
+#### (b)
+The worst possible running time is O(n^2) because the program might have to run to full length linear searches to find a duplicate ( the duplicate items are the last 2 items)
+
+#### \(c\)
+The algorithm is still correct, as the second loop only scans the values prior to the item selected by the first loop.
+
+#### (d)
+The algorithm will run about twice as fast, as it is removing a lot of extra searches. It doesnt search every value against every value, it only searches previous values against each value - which cuts out a lot of extra searches at the start of the list
```

Comment:
... so the idea is that the initial algorithm ends up checking each pair twice (i.e. checking both (a,b) and (b,a)), whereas this one checks each pair only once.